pyrit/memory/memory_interface.py

```diff
@@ -283,17 +283,18 @@ def get_prompt_ids_by_orchestrator(self, *, orchestrator_id: str) -> list[str]:
 
         return prompt_ids
 
-    def duplicate_conversation_for_new_orchestrator(self, *, new_orchestrator_id: str, conversation_id: str) -> str:
+    def duplicate_conversation(self, *, conversation_id: str, new_orchestrator_id: str = None) -> str:
         """
-        Duplicates a conversation from one orchestrator to another.
+        Duplicates a conversation for reuse
 
         This can be useful when an attack strategy requires branching out from a particular point in the conversation.
         One cannot continue both branches with the same orchestrator and conversation IDs since that would corrupt
         the memory. Instead, one needs to duplicate the conversation and continue with the new orchestrator ID.
 
         Args:
-            new_orchestrator_id (str): The new orchestrator ID to assign to the duplicated conversations.
             conversation_id (str): The conversation ID with existing conversations.
+            new_orchestrator_id (str, optional): The new orchestrator ID to assign to the duplicated conversations.
```

Comment:
```suggestion
            new_orchestrator_id (str, Optional): The new orchestrator ID to assign to the duplicated conversations.
```
jupyterbook doesn't like this in lowercase (it warns, so that's why you don't see an error)